src/main/java/frc/robot/systems/EveryBotIntakeFSM.java

```diff
@@ -0,0 +1,340 @@
+package frc.robot.systems;
+// WPILib Imports
+import edu.wpi.first.wpilibj.Timer;
+
+
+
+// Third party Hardware Imports
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+// Robot Imports
+import frc.robot.TeleopInput;
+import frc.robot.HardwareMap;
+
+public class EveryBotIntakeFSM {
+	/* ======================== Constants ======================== */
+	// FSM state definitions
+	public enum EveryBotIntakeFSMState {
+		INTAKING,
+		IDLE_FLIPCLOCKWISE,
+		IDLE_FLIPCOUNTERCLOCKWISE,
+		IDLE_STOP,
+		OUTTAKING
+	}
+	// Distance definitions
+	public enum ItemType {
+		CUBE,
+		CONE,
+		EMPTY
+	}
+	// hello
+	//HAVE TO CHANGE BASED ON TEST
+	private static final double KEEP_SPEED = 0;
+	private static final double INTAKE_SPEED = 0.1;
+	private static final double RELEASE_SPEED = -0.07; //DONT FORGET -
```

Comment:
Don't forget what?

Reply:
Signifies to not forget the negative sign (-) in the release speed constant.